### ISSUE TITLE
train.py stops without errors

### ISSUE THREAD

**Issue by GiacomoPorpiglia** [glinscott/nnue-pytorch#260]
Hello. I have just downloaded the project and tried to run it. But when I run it with the current recommended training invocation I get the following output: 

Feature set: HalfKAv2_hm^
Num real features: 22528
Num virtual features: 768
Num features: 23296
Training with data.bin validating with val.bin
Global seed set to 42
Seed 42
Using batch size 16384
Smart fen skipping: True
WLD fen skipping: True
Random fen skipping: 3
Skip early plies: -1
Param index: 0
limiting torch to 2 threads.
Using log dir logs/
ModelCheckpoint(save_last=True, save_top_k=-1, monitor=None) will duplicate the last checkpoint saved.
C:\pythonEnvironments\TorchEnv\lib\site-packages\pytorch_lightning\trainer\connectors\accelerator_connector.py:467: LightningDeprecationWarning: Setting `Trainer(gpus=1)` is deprecated in v1.7 and will be removed in v2.0. Please use `Trainer(accelerator='gpu', devices=1)` instead.
  rank_zero_deprecation(
GPU available: True (cuda), used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
HPU available: False, using: 0 HPUs
Using c++ data loader
C:\pythonEnvironments\TorchEnv\lib\site-packages\pytorch_lightning\loops\utilities.py:94: PossibleUserWarning: `max_epochs` was not set. Setting it to 1000 epochs. To train without an epoch limit, set `max_epochs=-1`.
  rank_zero_warn(
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
Ranger optimizer loaded.
Gradient Centralization usage = False

   | Name         | Type                          | Params

0 | input        | DoubleFeatureTransformerSlice | 47.9 M
1 | layer_stacks | LayerStacks                   | 303 K

48.2 M    Trainable params
0         Non-trainable params
48.2 M    Total params
192.808   Total estimated model params size (MB)

After this output, the program just stops. What is the issue?

**Comment by avivmel** [comment]
How did you fix the problem?
